Taskfile.dist.yaml

```diff
@@ -35,3 +280,427 @@ tasks:
     desc: "Check if yaml files are prettier-formatted."
     cmds:
       - prettier -c "**/*.yaml" "**/*.yml"
+
+  lint:golangci-lint:
+    desc: "Lint with golangci-lint"
+    dir: $PWD
+    cmds:
+      - golangci-lint run ./... --config {{.GITHUB_WORKSPACE}}/.golangci.yaml
+
+  format:
+    desc: 'Run all code formatters. Important vars: "paths".'
+    run: once
+    cmds:
+      - format:client
+      - format:e2e
+      - format:server
+
+  format:client:
+    desc: "Format client code"
+    cmds:
+      - gci write -s Standard -s Default -s 'Prefix(github.com/werf)' {{.paths | default "client/"}}
+      - gofumpt -extra -w {{.paths | default "client/"}}
```

Comment:
Move this to the service task, let's call it `_format`. `.paths` specified explicitly, no defaults.